lib/analyzer/applications/common.ts

```diff
@@ -0,0 +1,79 @@
+import * as path from "path";
```

Comment:
Could you maybe rename this from common.ts to something specific to code lineage? I'm thinking just so people on my team in the future know that this code is "owned" by a different team.

Reply:
Good point